.markdownlint.jsonc

```diff
@@ -0,0 +1,26 @@
+{
+  // Enable all markdownlint rules
+  "default": true,
+
+  // Enable line length check but exclude tables and code blocks
+  "MD013": {
+    "line_length": 120,
+    "tables": false,
+    "code_blocks": false
+  },
+
+  // Allow multiple top level headings
+  "MD025": false,
+
+  // Allow multiple headings with the same content
+  "MD024": false,
+
+  // Exclude code block style
+  "MD046": false,
+
+  // Allow inline HTML
+  "MD033": true,
+
+  // Allow no heading at start of file
+  "MD041": false
```

Comment:
I tried to adapt the rules to adhere to how things are set up now while also enforcing some consistency through our markdown files. I'd be happy to enforce even more "general guidelines" though if that it is OK with you.

MD025
Checked and no files have this apparently. Though there were several.

MD024
This can be set the true if we use `allow_different_nesting` like you suggested.

MD046
This was added in error (remnant from other packages which linted TS/JS inside code blocks as well I think. I got confused.

MD033
The comment says allow but the boolean actually disallows it as the rule is enabled. My bad! This one can be deleted and left default.

MD041
`CODE_OF_CONDUCT.md`, `.github/PULL_REQUEST_TEMPLATE.md` and  `config/README.md` all start without a  `#h1` heading. Proposed solution:
- add exception / comment disable to PULL_REQUEST_TEMPLATE and CODE_OF_CONDUCT
- Change config README to have an h1 heading

Would then change config to 
```jsonc
{
  // Enable all markdownlint rules
  "default": true,

  // Enable line length check but exclude tables and code blocks
  "MD013": {
    "line_length": 120,
    "tables": false,
    "code_blocks": false
  },

  // Allow multiple headings with the same content
  "MD024": {
    "allow_different_nesting": true
  }
}
```

Reply:
You can also add a header to the code of conduct (just saying code of conduct). That way only the PR template would need to be ignored. The Issue templates also break this though, doesn't the entire github folder have to be ignored?

Reply:
Yeah they shouldn't now you mention it. For some reason the GitHub issue templates pass linting on that rule... I think because of the construct in the beginning of the file...

Ignoring everything in .github is valid or we use an extension of the base config there disabling MD041